script/bench.rb

```diff
@@ -182,17 +188,21 @@ def port_available?(port)
 puts "Populating Profile DB"
 run("bundle exec ruby script/profile_db_generator.rb")
 
-puts "Getting api key"
-api_key = `bundle exec rake api_key:create_master[bench]`.split("\n")[-1]
+puts "Getting admin api key"
+admin_api_key = `bundle exec rake api_key:create_master[bench]`.split("\n")[-1]
 
-def bench(path, name)
+puts "Getting user api key"
+user_api_key = `bundle exec rake user_api_key:create[user1]`.split("\n")[-1]
+
```

Comment:
If the db isn't empty for some reason (a plugin creates a user on seed on my case), it will fail silently here, so let's add a check for it?

```suggestion
raise "Failed to obtain an API Key" if user_api_key.empty?
```